src/install/local/cli-install.md

```diff
@@ -0,0 +1,50 @@
+# Install and manage local MDAI cluster with CLI
+
+Our CLI will help you install and manage MDAI clusters locally.  
+
+## Prerequisite
+Since we are using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) cluster you have to have a [Docker](https://docs.docker.com/engine/install/) installed.
```

Comment:
```suggestion
Since we are using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) cluster you have to have [Docker](https://docs.docker.com/engine/install/) installed.
```